src/app/user/artist/component/artist-list/artist-list.component.ts

```diff
@@ -51,19 +56,18 @@ export class ArtistListComponent implements OnInit {
   }
 
   filterByArtType(name: string) {
-    this.activeArtType = name;
-    this.artistService.filterByArtType(name);
+    this.artistList = this.filterService.activateArtTypeNameFilter(name);
   }
 
-  sortItemsByLargeFollowNumber() {
-    this.artistService.sortItemsByLargeFollowNumber();
+  sortArtistsByFollowerNumberDesc() {
+    this.artistList = this.filterService.sortArtistsByFollowerNumberDesc();
```

Comment:
why not to pass the list to the function ?

Reply:
It's already defined by the constructor @gh-assan